std_msgs/Header header
armor_vision/ArmorDetection[] detections

